source/app/shared/utilities.ts

```diff
@@ -23,7 +23,24 @@ function formatRoute(original: string, ...args: string[]): string {
 }
 
 function createPlayFabLink(titleId: string, uri: string, isReact: boolean): string {
-    return `https://developer.playfab.com/en-US/${isReact ? `r/t/` : ``}${titleId}/${uri}`;
+    const playFabMainProdUrl = ".playfabapi.com";
+
+    var urlRoot;
```

Comment:
Please use `let` and `const` as appropriate for TypeScript variables. This should be `let` and have a type, i.e. `let urlRoot: string;` and the variables that don't change inside the `else` on lines 36-38 should be `const`.